common/app/implicits/Requests.scala

```diff
@@ -22,14 +22,13 @@ trait Requests {
 
     lazy val isRss: Boolean = r.path.endsWith("/rss")
 
-    lazy val isAmp: Boolean = r.path.endsWith(AMP_SUFFIX)
+    lazy val isAmp: Boolean = r.getQueryString("amp").isDefined || r.host == Configuration.amp.host
```

Comment:
`isAmp` is true if the amp query param is defined and equal to `1`? Not sure it matters though…